[RHACS] ROX-18672: Adding documentation for Compliance v2 and UI changes to compliance v1 (#71171)
openshift/openshift-docs: modules/understanding-the-cluster-compliance-page.adoc

// Module included in the following assemblies:
//
// * manage-compliance/managing-compliance-20.adoc

:_mod-docs-content-type: CONCEPT
[id="understanding-the-cluster-compliance-page_{context}"]

> = Understanding the cluster compliance page

Author: /s Overview of the cluster compliance page

WDYT?

> = Understanding the cluster compliance page

Author: @kerrycar

> = Understanding the cluster compliance page

Reviewer: Or "Cluster compliance page overview" , either

The cluster compliance page organizes information in the following groups:

* *Cluster*: Gives the details of your cluster and provides a snapshot of its current state and configurations.

> * *Operator status*: Assesses the health and operational status of the operators within your cluster and ensures that the operators are running optimally and functioning seamlessly.

Reviewer 2: ```suggestion
* *Operator status*: Assesses the health and operational status of the Compliance Operator instance within your cluster and ensures that the operator is running optimally and functioning seamlessly.
```

// Module included in the following assemblies:
//
// * manage-compliance/managing-compliance-20.adoc

:_mod-docs-content-type: CONCEPT
[id="understanding-the-cluster-compliance-page_{context}"]
= Understanding the cluster compliance page

The cluster compliance page organizes information in the following groups:

* *Cluster*: Gives the details of your cluster and provides a snapshot of its current state and configurations.
* *Operator status*: Assesses the health and operational status of the operators within your cluster and ensures that the operators are running optimally and functioning seamlessly.

> * *Build date*: Provides important information about the build date and provides a timestamp for the last configuration update or change to your cluster.

Reviewer 2: ```suggestion
```
